src/Component/src/State/Provider/SecurityAttributeProviderInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\State\Provider;
+
+use Sylius\Resource\Context\Context;
+use Sylius\Resource\Metadata\Operation;
+
+/**
+ * @experimental
+ */
+interface SecurityAttributeProviderInterface
+{
+    public function getAttribute(Operation $operation, Context $context): array|string;
```

Comment:
To be equals to the Symfony argument name
```suggestion
    public function getAttributes(Operation $operation, Context $context): array|string;
```